docs/extending/custom_image_filters.md

```diff
@@ -36,3 +36,17 @@ Use the filter in a template, like so:
 
 {% image page.photo width-400 blur-7 %}
 ```
+
+If your custom image filter depends on fields within the ``Image``, for instance those defining the focal point, add a ``vary_fields`` property listing those field names to the subclassed ``FilterOperation``. This is so that a new image rendition is created when any of these fields are changed:
```

Comment:
Nitpick, and maybe it is me, but I had to study the last line to get the exact meaning.

> This is so that a new image rendition is created when any of these fields are changed:

It think this is simpler:

> This ensures that a new rendition is created whenever the focal point is changed: